query_timetable.py

```diff
@@ -0,0 +1,34 @@
+import json
```

Comment:
Maybe add a shebang to the first line of the file so you can run this file directly from terminal without having to call `python3 /path/to/file/query_timetable.py` just call `/path/to/file/query_timetable.py` which would also allow you to add this to the PATH variable on the server so you can just call `query_timetable args` from anywhere (supposing we drop the .py extension otherwise it would be `query_timetable.py args` form anywhere.
```suggestion
#!/usr/bin/python3
import json
```

Reply:
Yeah that's fine!  The reason I don't usually put shebangs on python scripts is because I generally prefer calling python explicitly: that way, no matter where people put the executables, or which exact version of python they use it's fine either way.  That being said, it doesn't hurt at all, so I'm gonna commit this change.